ee/clickhouse/queries/breakdown_props.py

```diff
@@ -91,6 +101,22 @@ def get_breakdown_prop_values(
     )[0][0]
 
 
+def _to_value_expression(
+    breakdown_type: Union[Literal["event", "person", "cohort", "group"], None],
+    breakdown: Union[str, List[Union[str, int]], None],
+    breakdown_group_type_index: Optional[int],
+) -> str:
+    if breakdown_type == "person":
+        return get_single_or_multi_property_string_expr(breakdown, "person", "person_props", "value")
+    elif breakdown_type == "group":
+        value_expression, _ = get_property_string_expr(
+            "groups", cast(str, breakdown), "%(key)s", f"group_properties_{breakdown_group_type_index}"
+        )
+        return f"{value_expression} AS value"
```

Comment:
non-blocking

is it worth removing the identifier param to `get_single_or_multi_property_string_expr` and explicitly doing `AS value`. I feel that's a little less indirect about what the `identifier` param is for.

Reply:
Sometimes it is "value" and sometimes "props"

Reply:
I renamed it query_alias to see if that is clearer